feature-libs/organization/order-approval/assets/translations/en/order-approval.i18n.ts

```diff
@@ -17,6 +17,20 @@ export const orderApprovalDetails = {
       placeholder: '',
     },
   },
+  permissionResults: {
+    header: 'Customer Approval Details',
+    permission: 'Permission',
+    approver: 'Approver',
+    status: 'Status',
+    approverComments: 'Approver Comments',
+    noApprovalNotes: 'None',
```

Comment:
Just for consistency sake with `approverComments`, should this be `noApprovalComments` instead of `noApprovalNotes`

Reply:
Done.